Examples/ArcGISToolkitExamples/JobManagerExample.swift

```diff
@@ -293,13 +299,17 @@ class JobManagerExample: TableViewController {
             
             // register the job with our JobManager shared instance
             JobManager.shared.register(job: job)
+
+            // try to keep the app running in the background for this job if possible
+            let backgroundTaskIdentifier = self.startBackgroundTask()
```

Comment:
Consider starting the background task as soon as this method is triggered rather than waiting for `load` to finish. Since `load` is network dependent, it's possible the app goes to the background _before_ the load is done. If it does then this background task will never get started. The user likely doesn't know (nor should they) if the task is still loading or not.

Reply:
I did consider that. I agree with the benefits. A few cons perhaps:

- I would now need to make sure to call `endBackgroundTask` anytime it's short-circuited.
- I could not move the code into the job manager, as @mhdostal suggested

Reply:
fixed